### DISCUSSION TITLE
CPU performance - readme.md

### DISCUSSION THREAD

**Discussion by palladium123** [SYSTRAN/faster-whisper#155] [General]
Just out of curiosity, when you ran your CPU benchmarks, presumably you ran on 8 physical threads on the Xeon 6226R (as opposed to a 4C/8T config)? Also was AVX512 enabled?

**Comment by guillaumekln** [comment]
This CPU has 16 physical cores but the benchmark was run on a cloud instance, so there is no guarantee that the treads are mapped to physical cores.

AVX512 was enabled.
